app/helpers/checkouts_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Helper for checkouts views
+module CheckoutsHelper
+  def list_group_item_status_for_checkout(checkout)
+    if checkout.recalled?
+      'list-group-item-danger'
+    elsif checkout.overdue?
+      'list-group-item-warning'
+    end
+  end
+
+  def today_with_time_or_date(date, short_term: false)
+    return l(date, format: :short) unless short_term
+    return l(date, format: :short) unless date.today?
+
+    "Today at #{date.strftime('%l:%M%P')}"
```

Comment:
Should we add this as another localization format (`time_only` or some such?) 🤷‍♂ ?

Reply:
I embarrassingly didn't realize we could define our own.  lol, good to know.